test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,77 @@
+require('dotenv').config();
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'Ktagudelo';
+const repositories = 'repos';
+const repositoryName = 'Proyecto-marvel';
+
+describe('GitHub User Create And Modify Issue With A Title', () => {
+  it('Consume GET GitHub User Logged', async () => {
+    const response = await axios.get(`${urlBase}/user`, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
```

Comment:
to avoid having to send the header in each request, you can create an instance, store it in a global variable and initialize it with the header.
```js
const instance = axios.create({
  headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` }
});
```

Then you call the instance object in each test without the need to declare the header